src/Nordea/Components/Card.elm

```diff
@@ -53,6 +54,7 @@ import Nordea.Resources.Icons as Icons
 
 type alias CardProperties msg =
     { title : Maybe String
+    , htmlTitle : Maybe (Html msg)
```

Comment:
What about reducing the setTitle function to set the string title as a htmlTitle? Then we avoid state

Reply:
i was thinking about it. I guess it depends on how the gap-stuff is dealt with, to avoid UI-breakage for all other users.